keyboards/crkbd/keymaps/rjhilgefort/readme.md

```diff
@@ -0,0 +1,117 @@
+                                        KC_TRNS, _______, _______,       KC_0,   MO(5), _______
+                                    //`--------------------------'  `--------------------------'
+```
+
+#### Keymap 4: Media & Numbers (Linux)
+
+```
+  ,-----------------------------------------------------.                    ,-----------------------------------------------------.
+    _______, _______, KC_BRID, KC_VOLU, KC_BRIU, _______,                      _______,    KC_7,    KC_8,    KC_9, KC_ASTR, _______,
+  |--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+    _______, KC_MUTE, KC_MPRV, KC_MPLY, KC_MNXT, _______,                      _______,    KC_4,    KC_5,    KC_6, KC_PLUS, _______,
+  |--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+    _______, _______, _______, KC_VOLD, _______, _______,                      _______,    KC_1,    KC_2,    KC_3, KC_BSLS, _______,
+  |--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                        KC_TRNS, _______, _______,       KC_0,   MO(5), _______
+                                    //`--------------------------'  `--------------------------'
+```
+
+#### Keymap 5: Extra
+
+```
+  ,-----------------------------------------------------.                    ,-----------------------------------------------------.
+    _______,   DF(0),   DF(1), _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
+  |--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+    _______, _______, _______, _______, _______, _______,                      KC_HOME, KC_PGDN, KC_PGUP,  KC_END, _______, _______,
+  |--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+    _______, _______, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
+  |--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                        KC_TRNS, _______, _______,    _______, KC_TRNS, _______
+                                    //`--------------------------'  `--------------------------'
+```
+
+## Building And Flashing
+
+Note, you'll see that most `crkbd`'s recommend a target of `avrdude` because most kits come with a Pro Micro. I put a Elite-C on mine and thus the target is `dfu`. You can see this in the [rules.mk](./rules.mk) as the `BOOTLOADER` configuration.
+
+#### CLI
+
+```
+sudo make crkbd:rjhilgefort:flash
```

Comment:
In general we recommend against the use of sudo - set up the [udev rules](https://docs.qmk.fm/#/faq_build?id=linux-udev-rules) if flashing is not working for you.

Reply:
got it, thank you for the tip!

Reply:
```suggestion
make crkbd:rjhilgefort:flash
```